webrender/res/ps_gradient.fs.glsl

```diff
@@ -11,10 +11,5 @@ void main(void) {
     vec2 local_pos = vPos;
 #endif
 
-    alpha = min(alpha, do_clip(local_pos));
     oFragColor = vColor * vec4(1, 1, 1, alpha);
-
-#ifdef WR_FEATURE_TRANSFORM
```

Comment:
@glennw 
Looks like we were multiplying the alpha twice. I assume this was a mistake.